### DISCUSSION TITLE
Dependency on mmcv

### DISCUSSION THREAD

**Discussion by aswanthkrishna** [open-mmlab/mmdeploy#1248] [Q&A]
Is it possible to deploy a mmdetection model without dependency on mmcv?

**Comment by lvhan028** [comment]
mmcv is a must for mmdetection.
Does mmcv bring any inconvenience for you?

**Reply by aswanthkrishna** [reply]
mmcv have some license issues (https://github.com/open-mmlab/mmcv/blob/master/LICENSES.md) that is keeping me from deploying my mmdetection trained models in production at my organisation. I was looking at ways i can ship mmdet models without using mmdet/mmcv codebase in it.

**Reply by lvhan028** [reply]
You can try to deploy mmdet models with specific inference engines, like tensorrt, openvino, etc. according to you target platform and device.

**Reply by lvhan028** [reply]
Model converter in MMDeploy can help with converting mmdetection models to specified inference engine models. 
After that you can use inference SDK from MMDeploy to do model inference.
Can this solution meet your requirements?
